entity-framework/core/querying/related-data.md

```diff
@@ -51,6 +51,27 @@ You may want to include multiple related entities for one of the entities that i
 > [!CAUTION]
 > Since version 3.0.0, each `Include` will cause an additional JOIN to be added to SQL queries produced by relational providers, whereas previous versions generated additional SQL queries. This can significantly change the performance of your queries, for better or worse. In particular, LINQ queries with an exceedingly high number of `Include` operators may need to be broken down into multiple separate LINQ queries in order to avoid the cartesian explosion problem.
 
+### Filtered include
+
+> [!NOTE]
+> This feature will be introduced in EF Core 5.0.
```

Comment:
Just write is....

Reply:
Also it is already introduced. It is that 5.0 is not released yet. so will be would be incorrect. :trollface:

Reply:
I am very tempted to launch into this discussion! But I am strong.